packages/relay/src/lib/clients/cache/ICacheClient.ts

```diff
@@ -19,16 +19,21 @@
  */
 
 export interface ICacheClient {
-  get(key: string, callingMethod: string, requestIdPrefix?: string): any;
-  set(key: string, value: any, callingMethod: string, ttl?: number, requestIdPrefix?: string): void;
-  delete(key: string, callingMethod: string, requestIdPrefix?: string): void;
-  clear(): void;
-  multiSet(keyValuePairs: Record<string, any>, callingMethod: string, requestIdPrefix?: string): void;
+  get(key: string, callingMethod: string, requestIdPrefix?: string | undefined): Promise<any>;
```

Comment:
The presence of `?` already signifies that `requestIdPrefix` could be `undefined`.